frontend/components/styles/sizes.ts

```diff
@@ -1,5 +1,5 @@
 export const NAV_HEIGHT = '4.25rem'
-export const NAV_WIDTH = '14%'
+export const NAV_WIDTH = '12rem'
 export const MAX_BODY_HEIGHT = `calc(100vh - ${NAV_HEIGHT})`
```

Comment:
think NAV_WIDTH is unused now. But also should we even have this `sizes.ts`? Seems wrong to me given we r using tailwind now?

Reply:
Yeah, we shouldn't have this anymore, this is more of a gradual change. I might still keep a constants file for various purposes.